app/src/store/playgroundStore.tsx

```diff
@@ -6,6 +6,20 @@ export type GenAIOperationType = "chat" | "text_completion";
 let playgroundInstanceIdIndex = 0;
 let playgroundRunIdIndex = 0;
 
+/**
+ * Generates a new playground instance ID
+ */
+export const generateInstanceId = () => playgroundInstanceIdIndex++;
+
+/**
+ * Resets the playground instance ID to 0
+ *
+ * NB: This is only used for testing purposes
+ */
+export const resetInstanceId = () => {
+  playgroundInstanceIdIndex = 0;
+};
+
```

Comment:
```suggestion
export const _resetInstanceId = () => {
  playgroundInstanceIdIndex = 0;
};

```

just to make it weird to use